### PR TITLE
Add Module::custom_section_index helper

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: src/elements/module.rs [hunk 1]
```diff
@@ -109,6 +109,17 @@ impl Module {
 		Ok(())
 	}
 
+	/// Returns the index of a custom section, if found.
+	/// This will work seamlessly when some custom sections are parsed.
+	pub fn custom_section_index(&self, name: &str) -> Option<usize> {
+		self.sections().iter().position(|s| match s {
+			Section::Custom(_section) => _section.name() == name,
```
Comment (jakelang) [inline src/elements/module.rs]: We call a method on `_section` so we don't need the underscore, I think.

**Comment by NikVolf** [comment]
Thanks! Any reason to increase api surface with this rather specific query?

**Comment by axic** [comment]
We are using this in [wasm-chisel](https://github.com/wasmx/wasm-chisel) to remove sections. It has been maintained there for a while, but the fact parity-wasm replaces custom sections with new enum types when parsed makes our code tightly integrated to every version bump.

**Comment by NikVolf** [comment]
@axic Sure Alex, but it's really out of scope

being a low-level format library, parity-wasm should allow you to do anything with wasm,  but with minimum api surface to do that anything

We have a more high-level api in here, which also tracks internal references to keep the wasm consistent (which parity-wasm is totally ignores):
https://github.com/paritytech/wasm-utils/blob/master/src/graph.rs

Let me know if you might be interested in such api, I can split it off in separate lib!

**Pull request closed without merging**